File: mixedProblems/problem_11.py
# Problem 10 :How to check if a given number is an Armstrong number ?
text = input("Enter your number: ")
number = int(text)
numberArray = []
for i in text:
    numberArray.append(int(i))

sum=0
count=1
for x in numberArray:
    for y in range(3):
        count = count*x

    sum = sum+count
    count=1

if(sum==number):
    print("The Number is an Armstrong number")
else:
    print("The Number is not an Armstrong number")

File: mixedProblems/problem_11.js
// Problem 10 :How to check if a given number is an Armstrong number ?
var number = 153
var arrayOfNumber = number.toString().split('')
sum = 0
for(var i =0;i<arrayOfNumber.length;i++){
    var n = parseInt(arrayOfNumber[i])
    var cubeOfNumber = n*n*n
    sum+=cubeOfNumber
}
if(sum==number){
    console.log("The number is Armstrong number")
}else{
    console.log("The number is not Armstrong number")
}